### ISSUE TITLE
strange issue with node.exe and Less

### ISSUE THREAD

**Issue by iansebryk** [crazy-max/neard#333]
i built a setup in SublimeText with less2css.  when i ran it, i got this error:

![sublimetext3-less2css-neard-error](https://user-images.githubusercontent.com/1943574/30722901-e4fee112-9ef0-11e7-94b4-092711011d6c.jpg)

strange thing is that node.exe runs just fine from any command line, cmd or PowerShell, so i do not know what the heck is going on.  everything is in the System PATH and/or User PATH.  😕 

i know this also might be a less2css issue, and i've got [an issue posted](https://github.com/timdouglas/sublime-less2css/issues/169) there as well, but since it looks like a neard thing, i'm posting here to find out what exploded and how to fix it.  💔

and [here](https://drive.google.com/file/d/0B8YzV8tK9yq6NFRGWi1COF85dDA/view?usp=sharing) are the logs.

**Comment by crazy-max** [comment]
Hi @iansebryk,

I think the PATH variable is overrided by Sublime Text.
You can use the app ProcessHacker to look at the environment variables for the Sublime Text process and check if Node.js is in the PATH (example here with the httpd process of Neard) :

![image](https://user-images.githubusercontent.com/1951866/30773165-354379dc-a06b-11e7-814e-bfd60856dcbf.png)

**Comment by iansebryk** [comment]
hmm... i shall check on this... get back to you soon...

**Comment by crazy-max** [comment]
Coincidence @iansebryk! I updated a few days ago the README and the website homepage to explain the term Neard precisely!

For your issue, you can create a file called `sublime_text.bat` next to the .exe with this content :

```bat
@ECHO OFF
set PATH=%NEARD_BINS%;%PATH%
start sublime_text.exe
```

Then run `sublime_text.bat`

**Comment by iansebryk** [comment]
you want me to actually READ??  lol.  😄 it has been a loooong time since i hit the neard.io site.  didn't exist when i started using it.

at any rate, i tried the script and.... no soap. 💔 still throws that error.  can i assume that it was intended to be copied verbatim and not with substitutions?

**Comment by crazy-max** [comment]
RTFM you know lol 😄
Yep verbatim copy if `sublime_text.exe` exists. If it does not work, maybe there are some args to pass to sublimetext for this kind of problem.

**Comment by iansebryk** [comment]
i know this is closed, but i wanted to quickly revisit this and clearly show the solution that finally worked properly (and i fully accept the RTFM dig....  😆 ).  and yes, i have extra time to do this stuff while i wait for another issue to get sorted out... ahem...  😉 

this instruxion was NOT on the github page for Less2CSS, i found it on the page for, of all things, HTML Prettify.  when you specify the path for Windoze *in SublimeText*, the slashes (while defaulting to Windoze system backslashes) **MUST BE CHANGED TO FORWARD SLASHES**.  once this is done, boop, everything works beautifully.
